{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2021-28378",
    "ASSIGNER": "cve@mitre.org"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "CWE-79"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "https://github.com/go-gitea/gitea/pull/14898",
        "name": "https://github.com/go-gitea/gitea/pull/14898",
        "refsource": "MISC",
        "tags": [
          "Patch",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://blog.gitea.io/2021/03/gitea-1.13.4-is-released/",
        "name": "https://blog.gitea.io/2021/03/gitea-1.13.4-is-released/",
        "refsource": "MISC",
        "tags": [
          "Release Notes",
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://github.com/PandatiX/CVE-2021-28378",
        "name": "https://github.com/PandatiX/CVE-2021-28378",
        "refsource": "MISC",
        "tags": [
          "Exploit",
          "Third Party Advisory"
        ]
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "Gitea 1.12.x and 1.13.x before 1.13.4 allows XSS via certain issue data in some situations."
      }
    ]
  }
}